pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -14,12 +14,19 @@ import (
 )
 
 const (
-	PodSecurityCustomerType       = "PodSecurityCustomerEvaluationConditionsDetected"
-	PodSecurityOpenshiftType      = "PodSecurityOpenshiftEvaluationConditionsDetected"
-	PodSecurityRunLevelZeroType   = "PodSecurityRunLevelZeroEvaluationConditionsDetected"
-	PodSecurityDisabledSyncerType = "PodSecurityDisabledSyncerEvaluationConditionsDetected"
+	PodSecurityCustomerViolationType          = "PodSecurityCustomerEvaluationViolationConditionsDetected"
+	PodSecurityOpenshiftViolationType         = "PodSecurityOpenshiftEvaluationViolationConditionsDetected"
+	PodSecurityRunLevelZeroViolationType      = "PodSecurityRunLevelZeroEvaluationViolationConditionsDetected"
+	PodSecurityDisabledSyncerViolationType    = "PodSecurityDisabledSyncerEvaluationViolationConditionsDetected"
+	PodSecurityCustomerInconclusiveType       = "PodSecurityCustomerEvaluationInconclusiveConditionsDetected"
```

Comment:
Why not just 1 inconclusive type?

Reply:
We could do that if the distinction between types of namespaces is not as important for inconclusive determinations as it is for violations - no opposition here.

Reply:
I do like this better - consolidated to 1 inconclusive type